ui/server/index.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+module.exports = function(app, options) {
+  const globSync   = require('glob').sync;
+  const mocks      = globSync('./mocks/**/*.js', { cwd: __dirname }).map(require);
+  const proxies    = globSync('./proxies/**/*.js', { cwd: __dirname }).map(require);
+
+  // Log proxy requests
+  const morgan = require('morgan');
+  app.use(morgan('dev'));
+
+  mocks.forEach(route => route(app, options));
+  proxies.forEach(route => route(app, options));
+};
```

Comment:
You'll have to walk me through what's going on here. I'm assuming this is all in an effort to mock web sockets?

Reply:
yeah, this is boilerplate that I think just replicates what was previous set up in `.ember-cli`. I created #7465 so I can revisit this and maybe remove some parts. For instance, I suspect `mocks` are superfluous.